updated battery and tasker articles (#230)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/solving-problems/battery.md

:::

> On Android devices running OS 7 and earlier, built-in statistics often attributed high data and/or battery usage to AdGuard. This was because AdGuard accounted for all the traffic it filtered from different apps. As a result, AdGuard's portion of total data usage and battery consumption was overstated, while that of other apps was understated.

Reviewer: ```suggestion
On Android devices running OS 7 and earlier, built-in statistics often attributed high data and/or battery usage to AdGuard. This was because AdGuard counted all the traffic it filtered from various apps. As a result, AdGuard's share of total data and battery usage was overstated, while other apps were understated.
```

> However, with the OS 8, this scenario has improved. Now, the data reflected in Android's built-in statistics for data usage aligns closely with reality, though there are minor discrepancies in battery usage data.

Reviewer: ```suggestion
With OS 8, however, this scenario has improved. Now the data reflected in Android's built-in data usage statistics is very close to reality, although there are minor discrepancies in the battery usage data.
```

> Regardless, AdGuard users can always get a true picture of the situation on the *Battery usage* screen.

Reviewer: Redardless —> Nevertheless
